test/common/config/utility_test.cc

```diff
@@ -318,20 +318,21 @@ TEST(UtilityTest, TypedStructToStruct) {
   EXPECT_TRUE(Protobuf::util::MessageDifferencer::Equals(out, untyped_struct));
 }
 
+namespace {
```

Comment:
the whole file is in an unnamed namespace, no need extra one.

Reply:
done